dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -342,7 +343,7 @@ import SubmitButton from "@/components/forms/parts/SubmitButton.vue";
 import { FormKitNode } from "@formkit/core";
 import UploadReports from "@/components/forms/parts/UploadReports.vue";
 import { formatAxiosErrorMessage } from "@/utils/AxiosErrorMessageFormatter";
-import DataPointForm from "@/components/forms/parts/kpiSelection/DataPointForm.vue";
+import DataPointForm from "@/components/forms/parts/kpiSelection/DataPointFormEuTaxonomy.vue";
```

Comment:
Should we call this smth like "DataPointFormWithToggle" ?
That way you would know the specificication it has just by reading the name (rather then knowing for which framework it is used).

Reply:
Added a todo to remember this

Reply:
Leaving this open as reminder

Reply:
(Don't forget to rename the component name of the import too.  Currently it is still `DataPointForm`)

Reply:
I think now it is possible to rename it finally.

Reply:
Renamed to DataPointFormWithToggle

Reply:
I suggest naming it the same way as the underlying component.
I find it confusing if the import is named different than the underlying component.

Reply:
It is though
![image](https://github.com/d-fine/Dataland/assets/104447806/133b983c-1c00-48ba-8a6c-7a3655694c3a)

Reply:
Oh sorry! I think I haven't had pulled 😆

Reply:
👍